el/sclang-help.el

```diff
@@ -23,16 +23,29 @@
 (require 'sclang-util)
 (require 'sclang-interp)
 (require 'sclang-language)
-(require 'sclang-mode)
-(require 'sclang-vars)
+(require 'sclang-vars nil 'ignore-missing-file)
```

Comment:
Do you mean that `'ignmore-missing-file` makes this check optional?

Reply:
Yes, normally `(require 'a-missing-package)` would raise a file-missing error. The "truthy" value for the third argument just suppresses that error. `sclang-vars` will be present when compiled with cmake. It will be absent when installed from github / pkg manager.